std/unicode.zig

```diff
@@ -146,21 +166,12 @@ pub fn utf8Decode4(bytes: []const u8) Utf8Decode4Error!u32 {
     return value;
 }
 
+// TODO replace with something faster
```

Comment:
this is unhelpful.

Reply:
sorry, that came off as rude. i mean it would be better to link to some faster implementations, like you did in the C++ code, or maybe just say "optimize with SIMD" or some suggestion to make it faster instead of this vague comment.